### ISSUE TITLE
[Tutorials] Error in import in tutorials

### ISSUE THREAD

**Issue by crgimenes** [nasa/openmct#1206]
I'm trying to integrate my telemetry server with the OpemMCT.
For this I am following the tutorial at https://nasa.github.io/openmct/docs/tutorials/#step-2-expose-the-telemetry-dictionary

Using console.log I can see that the dictionary is being loaded and I see the menu tree in OpemMCT screen. But I do not see the data, and apparently the queries are not being made on the telemetry  server side.

Probably there was some change in the API and this tutorial outdated.

Additionally I do not have much experience in debug node.js and the system does not indicate any error on the console so I do not know what is wrong.

Can someone please try to reproduce the tutorial and see the if it works or maybe show me a way to debug the problem?

**Comment by crgimenes** [comment]
Complementing ...
For my tests I am using the tutorial server and not mine server, and I'm sure it is working because I could test using wscat -c ws://localhost:8081 and sending commands manually.

**Comment by crgimenes** [comment]
I finally found it!

in the ExampleTelemetryProvider.js file I changed
```
['./src/ExampleTelemetrySeries'],
```
to
```
['./ExampleTelemetrySeries'],
```

Now I can see the fake telemetry data in the browser.
There are still other small changes to be made, for example, change the glyph parameter for the new cssclass, and probably more that I did not notice.

**Comment by VWoeltjen** [comment]
Thanks for reporting this! Looks like #1207 fixes this; I'll keep this issue open until that has been merged.

**Comment by crgimenes** [comment]
Thank you @VWoeltjen!

**Comment by VWoeltjen** [comment]
Resolved by #1207
